app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,35 @@
+class MoviesController < ApplicationController
+  def index
+    results = MovieQuery.new(movies_params).results
+
+    options = {}
+    options[:include] = movies_params[:include].split(",")
+    render json: MovieSerializer.new(results, options).serialized_json
+  rescue StandardError => e
+    render json: { error: e.message, status: 400 }, status: :bad_request
```

Comment:
That's very generic error handling. Incompatibile with Rollbar and security risk on production